posthog/models/project_secret_api_key.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils import timezone
+
+from posthog.models.activity_logging.model_activity import ModelActivityMixin
+
+from .personal_api_key import hash_key_value
+from .utils import generate_random_token
+
+
+class ProjectSecretAPIKey(ModelActivityMixin, models.Model):
+    """"""
```

Comment:
Nit: Empty docstring here. Good place to add a brief description of what it's for and how it differs from `PersonalAPIKey`.